### PR TITLE
ref(icon-refactor): replace with IconCalendar

### PR DESCRIPTION
**Pull request by doralchan**
**Before:**
Screen Shot 2020-03-16 at 5 28 27 PM

**After:**
Screen Shot 2020-03-16 at 5 28 09 PM

### CODE REVIEW

File: src/sentry/static/sentry/app/icons/icon-calendar.svg [hunk 1]
```diff
@@ -1,11 +0,0 @@
-<svg
```
Comment (billyvg) [inline src/sentry/static/sentry/app/icons/icon-calendar.svg]: Can we make sure this isn't used in getsentry

File: src/sentry/static/sentry/app/views/settings/components/forms/datePickerField.jsx [hunk 2]
```diff
@@ -8,7 +8,7 @@ import styled from '@emotion/styled';
 
 import {inputStyles} from 'app/styles/input';
 import DropdownMenu from 'app/components/dropdownMenu';
-import InlineSvg from 'app/components/inlineSvg';
+import {IconCalendar} from 'app/icons/iconCalendar';
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/settings/components/forms/datePickerField.jsx]: We generally import from 'app/icons' I think, should we try to be consistent here?

**Review by billyvg — APPROVED** [review]

File: src/sentry/static/sentry/app/views/settings/components/forms/datePickerField.jsx (hunk 2 above)
Comment (doralchan) [reply src/sentry/static/sentry/app/views/settings/components/forms/datePickerField.jsx]: Yep - I can change it

File: src/sentry/static/sentry/app/icons/icon-calendar.svg (hunk 1 above)
Comment (doralchan) [reply src/sentry/static/sentry/app/icons/icon-calendar.svg]: Yep - this was not in getsentry

File: src/sentry/static/sentry/app/views/settings/components/forms/datePickerField.jsx (hunk 2 above)
Comment (billyvg) [reply src/sentry/static/sentry/app/views/settings/components/forms/datePickerField.jsx]: We seem to import using both ways pretty evenly, or rather, we only use  `app/icons` when importing multiple icons. This is a pretty easy refactor, so no need to change it atm.

**Review by dashed — APPROVED** [review]
Looks good to me 👍

**Pull request merged**